environment.yml

```diff
@@ -33,5 +33,6 @@ dependencies:
 - sqlalchemy~=2.0
 - tabulate<0.10.0,>=0.8.0
 - tqdm~=4.45
+- typing-extensions~=4.0
```

Comment:
it seems that it is not super nice that we do not have the python difference here but this seems like a general issue and not really urgent

Reply:
It is not very annoy. "only" takes 20 mins to wait to see the CIs run again.